src/user/user.service.ts

```
@@ -1,4 +1,60 @@
+    if (!useri) {
+      throw new NotFoundException('ID is not found');
+    }
+    return useri;
+  }
+
+  async findAll() {
+    return await this.prisma.user.findMany();
+  }
+
+  async createUser(RegisterUserDto: RegisterUserDto) {
+    return await this.prisma.user.create({
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)